packages/wonder-blocks-data/index.js

```diff
@@ -12,6 +14,19 @@ export const initializeCache = (source: ResponseCache): void =>
 export const fulfillAllDataRequests = (): Promise<ResponseCache> =>
     RequestTracker.Default.fulfillTrackedRequests();
 
+export const removeFromCache = <TOptions, TData>(
+    handler: IRequestHandler<TOptions, TData>,
+    options: TOptions,
+): boolean => ResCache.Default.remove<TOptions, TData>(handler, options);
+
+export const removeAllFromCache = <TOptions, TData>(
+    handler: IRequestHandler<TOptions, TData>,
+    predicate?: (
+        key: string,
+        cacheEntry: ?$ReadOnly<CacheEntry<TData>>,
+    ) => boolean,
+): number => ResCache.Default.removeAll<TOptions, TData>(handler, predicate);
```

Comment:
`TData` could be structured to include both the data being cached as well as metadata such as the last used timestamp right?

Reply:
What is in `TData` is entirely upto the IRequestHandler implementation, so yes, it can include whatever is needed by the use case being handled.